packages/nuxt-locokit/src/module.ts

```diff
@@ -149,29 +138,6 @@ export default defineNuxtModule<ModuleOptions>({
     //   })
     // }
 
-    /**
-     * Register all layouts
-     */
-    addLayout(
-      {
-        src: resolve(layoutsDir, './WithHeader.vue'),
```

Comment:
I didn't see change in file using layout. Is this normal?

Reply:
Yes.
Adding layout was clearly breaking the build and usage of the nuxt module, as it was referencing files in the module.
This seems not OK, but I didn't find any documentation on this.

Pages using layouts inside the module are OK.
Pages in the app using our `nuxt-locokit` module won't have access to our layouts, until our layouts need other files of the module (like paths).

So, for now, I don't have other work around for this. I simply remove the `addLayout`.
And this is fine as we were not using layouts in our nuxt playground / app.